operating/manage-compliance/managing-compliance-20.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-compliance-20"]
+= Managing compliance (2.0)
+include::modules/common-attributes.adoc[]
+:context: managing-compliance-20
+
+toc::[]
+
+By navigating to the Compliance (2.0) section in the {rh-rhacs-first} portal, you can seamlessly view, understand, schedule, and optimize the compliance scan status of your clusters.
```

Comment:
This might be a little misleading.  At the time of 4.4 this page will only allow customers to see compliance as it relates to their integration with the compliance operator.  It will only allow them to see results associated with the compliance operator.